docs/conf.py

```diff
@@ -68,13 +70,35 @@
     'jupyter_sphinx',
     'sphinx_autodoc_typehints',
     'reno.sphinxext',
+    'nbsphinx'
```

Comment:
Please also add this line:

```
"sphinxcontrib.jquery",  # Remove when changing html_theme to qiskit_ecosystem.
```

Reply:
Done in [3aace7c](https://github.com/Qiskit/qiskit-ibmq-provider/pull/1166/commits/3aace7ca1a738d29abac6172dfbccb581ba83129)